.github/checkgroup.yml

```diff
@@ -141,15 +137,15 @@ subprojects:
       - "setup.py"
       - ".actions/**"
     checks:
-      - "build-cuda (3.9, 1.10, 11.3.1)"
-      - "build-cuda (3.9, 1.11, 11.3.1)"
-      - "build-cuda (3.9, 1.12, 11.6.1)"
+      - "build-cuda (3.10, 1.11, 11.3.1)"
+      - "build-cuda (3.10, 1.12, 11.6.1)"
+      - "build-cuda (3.10, 1.13, 11.6.1)"
       - "build-hpu (1.5.0, 1.11.0)"
       - "build-ipu (3.9, 1.9)"
       - "build-NGC"
-      - "build-pl (3.9, 1.10, 11.3.1)"
-      - "build-pl (3.9, 1.11, 11.3.1)"
-      - "build-pl (3.9, 1.12, 11.6.1)"
+      - "build-pl (3.10, 1.11, 11.3.1)"
+      - "build-pl (3.10, 1.12, 11.6.1)"
+      - "build-pl (3.10, 1.13, 11.6.1)"
```

Comment:
These docker builds don't match with the changes added to the docker workflow.

Reply:
Good catch; could you send suggestions...

Reply:
Let me directly push a commit to fix this :)